tensorflow_lite_support/ios/task/processor/sources/TFLSegmentationResult.m

```diff
@@ -0,0 +1,80 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ ==============================================================================*/
+#import "tensorflow_lite_support/ios/task/processor/sources/TFLSegmentationResult.h"
+
+@implementation TFLCategoryMask
+@synthesize mask;
+@synthesize width;
+@synthesize height;
+
+- (id)copyWithZone:(NSZone *)zone {
+  TFLCategoryMask *categoryMask = [[TFLCategoryMask alloc] init];
+
+  [categoryMask setWidth:self.width];
```

Comment:
@priankakariatyml Please use property instead of calling the setter directly. Please fix other places in the rest of the code.
```
categoryMask.width = self.width;
categoryMask.height = self.height;
```

Reply:
Fixed